.github/workflows/build-simple-sample.yml

```diff
@@ -0,0 +1,39 @@
+name: Build simple sample
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+    name: Build simple sample
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./sample/simple
```

Comment:
We could alternatively use ./gradlew -p sample/simple ...` below and avoid the `../..`